Fix non-existence detection logic for NLB service/SKS Nodepool

falzm opened the pull request:
This change fixes a bug in the resource existence detection logic for
`exoscale_nlb_service` and `exoscale_sks_nodepool` resources, where
previously an error was returned if the parent resource (respectively
`exoscale_nlb` and `exoscale_sks_cluster`) was deleted outside of
Terraform instead of properly signal that the resource doesn't exist
anymore and prompt Terraform to re-create it.

Fixes #81.

bcachet commented:
Fixed with release: https://github.com/exoscale/pithos-private/releases/tag/0.8.8-SNAPSHOT
We can define CORS/ACL. Any modification is updated
If we set etag information of an object, it is uploaded if modified.

Commit:
Fix non-existence detection logic for NLB service/SKS Nodepool

This change fixes a bug in the resource existence detection logic for
`exoscale_nlb_service` and `exoscale_sks_nodepool` resources, where
previously an error was returned if the parent resource (respectively
`exoscale_nlb` and `exoscale_sks_cluster`) was deleted outside of
Terraform instead of properly signal that the resource doesn't exist
anymore and prompt Terraform to re-create it.

Fixes #81.
